apps/spruce/src/pages/task/taskTabs/TaskHistory/index.tsx

```diff
@@ -73,6 +81,8 @@ const TaskHistory: React.FC<TaskHistoryProps> = ({ task }) => {
   const { tasks = [] } = taskHistory ?? {};
 
   const groupedTasks = groupTasks(tasks, shouldCollapse);
+  const numVisibleTasks = Math.floor(width / (SQUARE_SIZE + SQUARE_BORDER * 2));
```

Comment:
`SQUARE_SIZE + SQUARE_BORDER * 2` is also defined in BuildRow. Does it makes sense to save as a constant somewhere?